Setting the scope in debian/maven.rules does not propagate the setting to
/usr/share/maven-repo/org/postgresql/postgresql/debian/postgresql-debian.pom,
so patch it manually here
Bug: #900615

--- a/pom.xml
+++ b/pom.xml
@@ -43,7 +43,8 @@
         <dependency>
             <groupId>com.ongres.scram</groupId>
             <artifactId>client</artifactId>
-            <version>2.1</version>
+            <version>debian</version>
+            <optional>true</optional>
         </dependency>
         <dependency>
             <groupId>se.jiderhamn</groupId>
